docs/en/createapps.md

```diff
@@ -121,31 +121,31 @@ To manage administrative rights for user groups, products, or product configurat
 
 * **[UM_Server]** is the UM API server: **https://usermanagement.adobe.io/v2/usermanagement/**
 * Replace **{orgId}** with your organization's unique ID, which looks like this: "12345@AdobeOrg".
-* Replace the **{...Id}** elements with the unique IDs assigned to user groups, products, and product configurations that are defined for you organization.
+* Replace the **{...Id}** elements with the unique IDs assigned to user groups, products, and product profiles that are defined for you organization.
 
 ***
 
 ### For More Information
 
 For detailed descriptions and examples of specific user-management operations, see the following pages.
 
-* For specific request and response syntax, see [Managing Users](api/ManageRef.md).
-* For detailed syntax of the JSON commands structure and user account operations, see [User Management Actions](api/ActionsRef.md).
+* For specific request and response syntax, see [Managing Users](api/user.html).
+* For detailed syntax of the JSON commands structure and user account operations, see [User Management Actions](api/ActionsRef.html).
 * For examples of user-management requests, see:
-  - [Create users](samples/SampleCreate.md)
-  - [Manage pending invites](samples/SampleInvites.md)
-  - [Update user information](samples/SampleUpdate.md)
-  - [Add and remove membership and admin rights in user groups](samples/SampleGroups.md)<br>
-  - [Add and remove entitlements through product configuration membership](samples/SampleGroups.md)
-  - [Remove users](samples/SampleRemove.md)
-  - [Perform multiple actions for one user](samples/SampleMultiActions.md)
-  - [Perform actions for multiple users](samples/SampleMultiUser.md)
+  - [Create users](samples/SampleCreate.html)
+  - [Manage pending invites](samples/SampleInvites.html)
+  - [Update user information](samples/SampleUpdate.html)
+  - [Add and remove membership and admin rights in user groups](samples/SampleGroups.html)<br>
+  - [Add and remove entitlements through product profile membership](samples/SampleGroups.html)
+  - [Remove users](samples/SampleRemove.html)
+  - [Perform multiple actions for one user](samples/SampleMultiActions.html)
+  - [Perform actions for multiple users](samples/SampleMultiUser.html)
```

Comment:
Same note for all of these.

Reply:
As noted previously, this does build and function in https://git.corp.adobe.com/pages/JIL-v2/umapi-docs/en/createapps.html.